src/workspaces/list/RecentlyViewedWorkspaces.ts

```diff
@@ -2,11 +2,12 @@ import _ from 'lodash/fp';
 import { Fragment, ReactNode, useEffect, useMemo, useState } from 'react';
 import { div, h, span } from 'react-hyperscript-helpers';
 import Collapse from 'src/components/Collapse';
-import { recentlyViewedPersistenceId, RecentlyViewedWorkspaceCard } from 'src/components/workspace-utils';
-import { workspaceSubmissionStatus } from 'src/components/WorkspaceSubmissionStatusIcon';
 import { getLocalPref, setLocalPref } from 'src/libs/prefs';
-import { WorkspaceWrapper as Workspace } from 'src/libs/workspace-utils';
+import { recentlyViewedPersistenceId } from 'src/workspaces/common/state/recentlyViewedWorkspaces';
+import { RecentlyViewedWorkspaceCard } from 'src/workspaces/list/RecentlyViewedWorkspaceCard';
 import { getWorkspace, persistenceId } from 'src/workspaces/list/WorkspacesList';
+import { workspaceSubmissionStatus } from 'src/workspaces/list/WorkspaceSubmissionStatusIcon';
+import { WorkspaceWrapper as Workspace } from 'src/workspaces/utils';
```

Comment:
Another general comment... I've seen this alias several times. Could/should we rename this type?

Reply:
I'll take a look, but this may be a bigger can of worms than I'm willing to get into. I've always thought that it would be better if folks used the name `WorkspaceWrapper` instead of `Workspace` because there was an attempt to differentiate.

Reply:
To be clear, I wasn't suggesting changing it as part of this PR (or even in the near term).

It seems like what's now called `WorkspaceWrapper` is what gets passed around most of the time, so it might be nice to shorten that to `Workspace`.